### PR TITLE
Removes roundstart singularity, adds roundstart supermatter, supermatter now collapes into singularity

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor and Jordie
add: Singularity containment has been replaced on box, meta, and delta with a supermatter room. The supermatter gives ample warning when melting down, so hopefully we'll see fewer 15 minute rounds ended by a loose singularity.
add: Supermatter crystals now collapse into singularities when they fail, rather than explode.
/:cl:

Title basically says it.

People are sick of unfixable, silent engine failures derailing rounds within 20 minutes. The supermatter is already designed to start causing drama in various degrees without falling into a binary contained/round over. There are warning systems in place and steps you can take to avert disaster.

However, as singularity is a classic and the most exciting destruction in the game, the supermatter will now collapse into a stage three singularity upon failure rather than exploding.

If this idea doesn't instantly start lynch mobs I'll go ahead and map a supermatter room into meta and box.

### CODE REVIEW

**Review by MrsTonedOne — CHANGES_REQUESTED** [review]
Fuck you for closing the emitters pr for this. put closes #24308 in your pr, and leave the emitters pr the fuck alone

**Comment by Kromgar** [comment]
This... this works. Nice job Kor

**Comment by optimumtact** [comment]
I'm backing this, the singulo is simply too destructive and has very engaging gameplay.

**Comment by anconfuzedrock** [comment]
where does it removee singulo?

**Comment by Robustin** [comment]
What happens to the Tesla?

I support harder engine sabotage but considering support for my "Announced when the PA is firing into an incomplete containment" PR is split 50/50 I'd be surprised there's enough support for "replaces instant release engines with fixed engine that announces how close it is to "releasing" down to the %". 

Regardless, frequent loose engines is bad for the game.

**Comment by Kromgar** [comment]
@anconfuzedrock 

Don't you mean into a goofball?

**Comment by optimumtact** [comment]
@Robustin in this case support is not going to be a deciding factor, although it will be taken into account.

We think that this is necessary for the balance and health of the game.

**Comment by KorPhaeron** [comment]
@Robustin 

It was completely redundant anyway ever since it got buffed to do just as much if not more station damage as the singularity, I guess they'll stay in cargo though

**Comment by Cheridan** [comment]
I like the sound of this. Some things to consider though:

- I assume tesla is going to get gone as well, considering that it's just a singulo reskin at this point
- Singularity Beacon traitor item will likely need to be tweaked as it'll be a dud buy or at least much harder to use effectively

**Comment by Repukan** [comment]
Will this be a filler for delamination cascade?

**Comment by KorPhaeron** [comment]
@Repukan 

I've never actually seen that happen, whats it like?

**Comment by Repukan** [comment]
I haven't seen it either, but as far as my friend who plays vg regularly says, the engine melts down, everything starts turning into super matter, and the crew has to run towards a portal to get transported to a safe beach. All traitors lose their objectives and it's changed to Survive, and centcomm gets destroyed since the universe in which the delamination happens is destroyed.

**Comment by optimumtact** [comment]
I don't think we have vg's delamination, it might have been added after we ported it

**Comment by KorPhaeron** [comment]
That sounds exciting

**Comment by Repukan** [comment]
I think the shards and crystals we have are different from /vg/ though?

**Comment by Kromgar** [comment]
Shit we need that delamination shit that sounds cool.


P

**Comment by PKPenguin321** [comment]
No kor you're supposed to tell them rule 10 not make a pr to cater to them

**Comment by KorPhaeron** [comment]
The round restarting every 20 minutes for the same exact reason is not exciting story telling, and I primarily view SS13 as a vehicle for exciting story telling at its best.

**Comment by optimumtact** [comment]
especially when you consider how mechanically boring the singulo is.

Oh push these four wall thingies on, turn this up to 3, fill these with plasma and leave it alone for the rest of the round once it gets up to size and you drop the pa back to zero.

**Comment by Incoming5643** [comment]
I hate that I don't hate this

**Comment by Shapsy** [comment]
This makes singulooses rarer and thus more interesting when they do happen, and provides more value to being an engineer and gives them more work to do. Win win.

**Comment by Robustin** [comment]
@Incoming5643 

Sums up my thoughts. SM is the safest of safe engines, I'd almost like to see the singularity automatically ramp up to a Supermatter Sing to make up for how much safer its going to be.

But I think whats paramount is less loose engine rounds, they suck and are probably the single greatest anti-fun mechanic when they happen in consecutive rounds.

**Comment by Shapsy** [comment]
The supermatter can be tweaked from there to be more fun in its operation once it's the only engine that the game has to be balanced around

**Comment by Metallusions** [comment]
Intriguing.

So, how many times has supermatter delamination been started and averted on Deltastation so far?

**Comment by ma44** [comment]
This doesn't fix anything if someone just buys a SM shard and emitter roundstart, then goes and sets it up to damage the shard enough to release a singularity

1. buy a fun emag to bypass the sm crate id restriction (6 tc spent)

2. get two brief cases of case (2 more tc spent)

3. Order emitter crate, ship it back out with cash

4. order SM, get emitter set up somewhere in maintenance idk

5. Haul crate to emitter location, turn it on

6. Enjoy your stage 3 singularity at nearly roundstart and still have 12 more TCs to buy whatever you want

**Comment by Iamgoofball** [comment]
Hey what the fuck don't touch my goofball

**Comment by optimumtact** [comment]
@ma44 what you just described is infinitely harder than releasing the singularity

**Comment by Repukan** [comment]
A super matter shard would just explode.

What we're talking about is the crystal

**Comment by ShadowDeath6** [comment]
>"removes roundstart singularity"
>no revert tag

**Comment by PKPenguin321** [comment]
The singularity it collapses into starts at like, stage 2 or 3, and not 1, right?

**Comment by ShadowDeath6** [comment]
it says stage three in the OP

**Comment by PKPenguin321** [comment]
Right, I'm illiterate, sorry.

**Comment by GunHog** [comment]
Can its radio be silenced by shutting down Tcomms? The supermatter is the toughest of engines to unleash. What are some effective ways to release the engine? What roles can still reasonably do it (Engineer, Atmos, Assistant, AI) and how?

**Comment by KorPhaeron** [comment]
@Iamgoofball 

Can always change the supermatter to melt down in a random way (explosion, singulo, goofball) or melt down in different ways based on whats been done to it in the round etc

**Comment by Iamgoofball** [comment]
random meltdowns is 👌

**Comment by Iamgoofball** [comment]
that'll prevent intentional meltdowns for singulo

**Comment by PKPenguin321** [comment]
Completely random sounds lame, should have some way to impact it

**Comment by Iamgoofball** [comment]
@PKPenguin321 
powergame for singulo every round so i dont have to set up for SM or tesla would happen

**Comment by PKPenguin321** [comment]
Who cares, that requires intentionally overloading the supermatter. Maybe make it always explode and then spawn an engine.

**Comment by Iamgoofball** [comment]
@PKPenguin321 
Then at what point does it just become
"throw SM out in the singulo containment and don't bother actually doing SM"

**Comment by Shadowlight213** [comment]
This looks fun

**Comment by RemieRichards** [comment]
I support this but I don't like the supermatter -> singulo thing, it's just not how these things work.
supermatter does not spontaneously a blackhole make!

**Comment by Arianya** [comment]
@RemieRichards If one assumes the SM is a hyper-dense crystal matrix, it makes vague, technobabbly sense that improper energizing of the matrix could cause a catastrophic implosion of the already very-dense crystal into a singularity.

Harder to explain it turning into a goofball, but heyo.

**Comment by RemieRichards** [comment]
No.

**Comment by jughu** [comment]
Supermatter should explode into shards causing them to fly in all directions like immovable rods

**Comment by Iamgoofball** [comment]
@RemieRichards please explain the science behind wizard

**Comment by RemieRichards** [comment]
Technomancers, that one's already been answered with the age old quote of "sufficiently advanced tech is indistinguishable form magic"

**Comment by Militaires** [comment]
@remierichards supermatter doesnt exist and singulos arent even black holes nor are they how black holes work

so whatever with this we never had good realism.

**Comment by RemieRichards** [comment]
@Militaires they are black holes, the centre of a black hole is its singularity, a point of extremely high density.

**Comment by Militaires** [comment]
no shit

@RemieRichards yeah but they're a very inaccurate depiction of a black hole to begin with so why bother with REALISM when the base shit is just "it's a black hole that sucks thing" which is fine in itself but we never simulated real fucking ergospheres, gravitational lenses, schwarzchild radii to begin with

**Comment by RemieRichards** [comment]
I never claimed otherwise, I just said supermatter doesn't make black holes.

**Comment by Militaires** [comment]
and people cant mold metal into different shapes with their hands, firesuits dont protect you from hard vacuum and gravity does not work as a single consistent vector

so why are you nagging about specifically this

**Comment by RemieRichards** [comment]
Because it's dumb, anyways I don't have to explain it to you I can just bully kor.

**Comment by Militaires** [comment]
if you arent fine with things being dumb why are you here.

**Comment by Cebasiz** [comment]
i don't like it. there are reasons the singulo gets free, and those reasons are not necessarily "the singulo is too hard to maintain"

the reasons are:
1. nobody with any experience plays engineering
2. engineering doesn't care
3. engineering leaves engineering ASAP because there is fuck all to do
4. ^ above means singulo sabotage is easy as fuck
5. nobody cares about intruders in engineering and/or there is never enough security to deal with the problem due to a whole slew of other problems

this "fixes" that by just ignoring and going around all of these problems. long term, making engineering actually something worth playing again should be the goal, not just fixing singularity killing muh station, even though that is annoying. i mean really, what's going to stop it just from running amok anyway? none of these problems go away just by changing all of the maps to use supermatter.

i guess no one wants to take responsibility for making engineering a real department. which is disappointing.

**Comment by Arianya** [comment]
I mean, part of the reason engineering is abandoned is because our 3 main power sources (singulo, tesla, solars) require no maintenance/interaction after being set up. If SM is a more involved power source, why not start there to give engineers a reason to stay in the department?

**Comment by XDTM** [comment]
If repairing/autismforts are so unfun why would players ever play drones? The issue is that engineers are not supposed to be in engineering, they're supposed to go out to build and repair. Having a SM to manage will give an engineer something to do in the department, especially if we add gas micromanagement for additional bonuses.

**Comment by Arianya** [comment]
>implying people play drones to repair shit

I can list the number of times I've seen a drone actually repairing shit on one hand. With no fingers.

**Comment by Swagile** [comment]
cebasiz hit the nail on the head

this "fix" will go through, and people will ignore the warning messages anyway or have to break into engineering to fix it themselves because there are no engineers anywhere to be seen as either they are too inexperienced to do anything, or they are out doing their own projects because engineering in of itself is boring as fuck

most players who "master" engineering either play engineer for the quick tools and materials round start, or go assistant / scientist for a more free / engaging experience that rewards them for their hard work

**Comment by XDTM** [comment]
Engineering's job is to repair and build, engine caretaking is only one facet of the job which is usually pointless since there's several ways to set a singulo loose unless you watch it 24/7, which just means not playing. Putting in an engine which can be interacted with post-setup will help the issue, since someone could stay around to fiddle with it.
Plus the supermatter requires time to loose and gives a warning before it's too late, meaning an eventual traitor must plan either a way to destabilize it very, very fast or fortify/defend the engine until it goes critical, which is more fun than snipping a wire and watching the round end.

**Comment by KorPhaeron** [comment]
@tgstation/commit-access I guess take a vote on this or something or otherwise reach some sort of consensus and then I'll either close it or do the mapping for it

**Comment by GunHog** [comment]
@KorPhaeron, you never answered my previous comment:

>Can its radio be silenced by shutting down Tcomms? The supermatter is the toughest of engines to unleash. What are some effective ways to release the engine? What roles can still reasonably do it (Engineer, Atmos, Assistant, AI) and how?

**Comment by KorPhaeron** [comment]
no idea

**Comment by KorPhaeron** [comment]
Anyone can probably still do it just by emagging into engineering and breaking things, but they'd need to deal with security showing up to stop them

**Comment by AnturK** [comment]
Looks good to me, even if it's not the neatest solution, it opens up a lot of engineering space so alternative engines can be floor/atmos based.

**Comment by Cheridan** [comment]
>Can its radio be silenced by shutting down Tcomms? 

Yes

**Comment by Durkel** [comment]
So how long will it be until the inevitable forum and github pr about the supermatter being too easy to sabotage and we replace it with just solars?

**Comment by KorPhaeron** [comment]
@Durkel 

Well singulo lasted 7 years in /tg/ now, so hopefully at least that long

**Comment by Kromgar** [comment]
@KorPhaeron Do you think the supermatter delam could be ported? Maybe have it be like a 5% chance so that way its pretty rare and cool event  to see

**Comment by CaptainChloroform** [comment]
I love the idea of the SMS replacing the singulo. I remember one round which was the record iirc that the singulo was released in less than 5 minutes in.

**Comment by Kromgar** [comment]
@CaptainChloroform try 30 seconds

**Comment by CaptainChloroform** [comment]
@Kromgar As I hate to say it, I completely believe you.

**Comment by Kromgar** [comment]
So now we will know if the engineers let the delam happen they are all incompetent

**Comment by ma44** [comment]
@Repukan This pr makes the shard collapse into a singularity

@optimumtact Moving two crates into maintenance isn't really that hard, you don't even have to rush it and wait for some people to die

It's a good way to get a singularity l00se on lowpop or late into the round without going into the engine room

**Comment by coiax** [comment]
Moving the singularity and tesla into fun/!!fun!! engineering projects is fine for me. The SM engine at least provides some form of grace before fucking up.

Would probably suggest making energy beacons cheaper if this change goes into pass.

But yes, :+1:

**Comment by Robustitron** [comment]
Loose singulos aside, SM engine is just more fun and should be the example for making engines fun to actually run and maintain.

Singulo/Tesla gameplay is just setting it up then leaving, that sucks.

The SM can be throttled up or down and fucked around with depending on whats going on with the station. Throttle it down when things get dangerous, up when you need the power or just want to show that you can do it (afaik there's no reward for maxing the output). When the SM engine is fucking up, you can fix it and turn things around before it explodes. 

SM engine actually kind of feels like you're running and maintaining a baby's-first reactor past just setting it up. I wish the Singulo/Tesla were overhauled to have at least a little bit of depth like that.

**Comment by SgtxRootbeer** [comment]
@robustitron
That actually reminds me of a design theory for a sub-critical thorium reactor...
Thorium isn't fissile enough to go critical like uranium or plutonium, but if you use a particle accelerator (not a ss13 one, this is irl science) you can make it react, and harvest power from it. If you keep feeding it with power, you get more power out, but, to shut it down, all you have to do is turn of your accelerator, and the reaction stops. 
The sm shard seems to have a larger root in IRL science than people think!

**Comment by optimumtact** [comment]
@KorPhaeron my vote is yes

**Comment by KorPhaeron** [comment]
@tgstation/commit-access 

Added the maps, this is ready whenever

**Comment by KorPhaeron** [comment]
I see thumbs up from Cheridan and Changelingrain and Remie and Antur both said yes (other than remie hating the lore) in the PR comments so I think we're actually unanimous

**Comment by lzimann** [comment]
Map edit on box: 
box

**Comment by Jordie0608** [comment]
Was that just copypasted in? There's quite a lot of errors and poor elements.
- Unterminated wires and piping (mostly the junctions)
- Double walls
- Underfloors in walls
- Those walls, tables and lockers
- Cabling changing color
- Vents and scrubbers over other underfloors
- Some strange placement of colored tiles
- Airlocks to now nowhere
- No wall objects
tl;dr This shouldn't be merged until the map is done properly; if you don't want to I can do it for you but I will call you a scrub many times

**Comment by KorPhaeron** [comment]
I'll just let tokiko do it I have no patience for mapping

**Comment by KorPhaeron** [comment]
@Jordie0608 you already think I'm a scrub so I'll take you up on the offer if tokiko doesn't end up doing it

**Comment by KorPhaeron** [comment]
I removed all the extra wires and pipes but some of them still overlap/are different colours

**Comment by KorPhaeron** [comment]
@Tokiko1 Will the supermatter actually power a full size station with a single emitter or do I absolutely need to add a laser room before this is merged

**Comment by SgtxRootbeer** [comment]
@KorPhaeron
I have gotten 24 megawatts outta a 3 emitter setup on Delta, you only really need 1 emitter to get power.

**Comment by KorPhaeron** [comment]
Cool, thanks, this should more or less be ready then

**Comment by lzimann** [comment]
New map picture:
new

**Comment by Tokiko1** [comment]
@KorPhaeron 
I'm actually going to download and test this right now. 1 emitter might be difficult, especially with a n2 setup.

The setup in the picture has 1 error though, being the filter in the bottom right corner. The filter is supposed to filter everything but the specified gas into space, however in this setup will actually move the filtered gas into space instead (unless there is some new way to setup filters to filter straight ahead instead of to the side).

**Comment by Robustin** [comment]
Just speaking from personal experience, if you can turn on the freezer you can prevent it from overloading. I've had a plasma fire raging inside the containment room and managed to stop the detonation just by turning on the freezer.

**Comment by MrStonedOne** [comment]
kor, if you could, could you maybe move the cooling loop to one of the sides, to make it easier for engis to create a laser room.

or at least move it some bit back.

**Comment by MrStonedOne** [comment]
@Tokiko1 you can make a pr to his pr that he can then merge.
